rllib/policy/dynamic_tf_policy.py

```diff
@@ -303,6 +310,17 @@ def __init__(
                     explore=explore,
                     is_training=self._input_dict.is_training,
                 )
+                if len(action_sampler_outputs) == 4:
+                    (
+                        sampled_action,
+                        sampled_action_logp,
+                        dist_inputs,
```

Comment:
should we not include this if statement and instead generically return the action sampler outputs? Users may want to expand the info that is returned by the action sampler -- I can't think of the usecase, but its possible CC @gjoliver

Reply:
that may be a bigger change compatibility-wise?
these data fields are actually immediately used below.